rpc/tester/tester_test.go

```diff
@@ -169,3 +169,12 @@ func TestEth_NewFilter(t *testing.T) {
 	err = code.UnmarshalJSON(rpcRes.Result)
 	require.NoError(t, err)
 }
+
+func TestEth_NewPendingTransactionFilter(t *testing.T) {
+	rpcRes, err := call(t, "eth_newPendingTransactionFilter", []string{})
+	require.NoError(t, err)
+
+	var code hexutil.Bytes
+	err = code.UnmarshalJSON(rpcRes.Result)
+	require.NoError(t, err)
```

Comment:
maybe continue this test and send a transaction then call eth_getFilterChanges and see if it shows up?

Reply:
well, yes... but this whole test suite need a overhaul anyways. So I was planning to do it in another ticket...

Reply:
besides, there is not even a test for send transaction yet, so perhaps we have to start with that first ?

Reply:
there's a function to deploy a test contract, you can use that or modify it to send a transaction

Reply:
could you please point me to where is it ?

Reply:
rpc/tester_test.go deployTestContract

Reply:
are you sure ? I cant find it:
https://github.com/ChainSafe/ethermint/search?q=deployTestContract&type=Code

Reply:
https://github.com/ChainSafe/ethermint/blob/development/rpc/tester/tester_test.go#L236

Reply:
repo search doesn't seems to work on forks

Reply:
ohhh this is new code and my branch was outdated. Thanks for the link, I will check now.

Reply:
test implemented, please re-check